bitsandbytes/autograd/_functions.py

```diff
@@ -230,6 +230,10 @@ def supports_igemmlt(device: torch.device) -> bool:
     nvidia16_models = ('GTX 1630', 'GTX 1650', 'GTX 1660')  # https://en.wikipedia.org/wiki/GeForce_16_series
     if any(model_name in device_name for model_name in nvidia16_models):
         return False  # these devices are technically cuda 7.5-capable, but they lack tensor cores
+    if device.type == "cpu":
```

Comment:
There are currently a couple of functions that support the CPU (block-wise quantization in 8-bit). How are these executed at the moment?

Reply:
This was thinking of CPU full functionality. Will remove this to avoid confusion.